oktapam/constants/descriptions/attributes.go

```diff
@@ -89,6 +90,8 @@ var (
 	LocalAccount                              = "The username of a local account on a server."
 	LocalAccounts                             = "The usernames of a local accounts on a server."
 	ManagedPrivilegedAccounts                 = "An array of managed accounts for password rotation."
+	MFAACRValues                              = "The authentication context class reference value used to defines a specific set of assurance level requirements that the protected resource requires."
+	MFAReAuthFrequencyInSeconds               = "The number of seconds a successful MFA is valid for before requiring another MFA to access the resource again."
```

Comment:
Matching this to the spec description. We could maybe trim it more if we wanted.
```suggestion
	MFAReAuthFrequencyInSeconds               = "The number of seconds an MFA verification remains valid. After this time users need to reauthenticate before they can open new connections to resources. Defining a value of `0` requires users to authenticate for every connection."
```